exec-core/src/main/java/com/capitalone/dashboard/exec/repository/PortfolioRepositoryThumbnail.java

```diff
@@ -0,0 +1,13 @@
+package com.capitalone.dashboard.exec.repository;
+
+import com.capitalone.dashboard.exec.model.PortfolioThumbnail;
+import org.bson.types.ObjectId;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface PortfolioRepositoryThumbnail extends CrudRepository<PortfolioThumbnail, ObjectId>{
+
+    List<PortfolioThumbnail> findAll();
```

Comment:
Is this method needed? `findAll()` is given by default, overriding is redundant

Reply:
We removed findAll() to avoid redundant and updated as findAllByNameAndThumbnail().